examples/widgets/msgbox/index.rst

```diff
@@ -5,4 +5,7 @@ Simple Message box
 .. lv_example:: widgets/msgbox/lv_example_msgbox_1
   :language: c
 
+.. lv_example:: widgets/msgbox/lv_example_msgbox_2
```

Comment:
We should add titles to the examples here.

Reply:
It seems that no widget doc has that. See any `docs/widgets/*.rst`.

Reply:
![Screenshot_20240731_233220](https://github.com/user-attachments/assets/363f7ada-d8e2-471c-a89c-ecc486333603)

Reply:
Ah yes, thanks, great catch. Please forgive me, I was looking at the wrong file.